docs/components/camera/_index.md

```diff
@@ -400,7 +408,6 @@ You can find additional assistance in the [Troubleshooting section](/appendix/tr
 ## Next Steps
 
 {{< cards >}}
-{{% card link="/services/vision" %}}
```

Comment:
I think let's keep that here regardless. Because the related service card doesn't explain what the service does